website/blog/2019-01-21-jest-24-new-website-babel-7-and-typescript-support.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Jest 24: New Website, Babel 7 and TypeScript support
+author: Simen Bekkhus
+authorURL: https://github.com/SimenB
+authorFBID: 100003004880942
+---
+
+Today we are happy to announce the next major release of Jest - version 24! It's been 4 months since the last release (and 8 since Jest 23), so this is a meaty one. Highlights include upgrading to Babel 7 across the board (bringing with it built-in support for testing TypeScript) fixing some long standing issues with missing console output and performance issues when computing large diffs, as well as a brand new sparkling website. ✨
+
+For a full list of all changes see the [changelog](https://github.com/facebook/jest/blob/master/CHANGELOG.md).
+
+<!--truncate-->
+
+## New Website
+
+[@orta](https://twitter.com/orta) has provided a beautiful redesign of Jest's website, which has been implemented by community members [@montogeek](https://twitter.com/montogeek) and [@brainkim](https://github.com/brainkim).
+
+The aim of the re-design was to highlight more of what makes Jest awesome, and to de-couple the idea that Jest is primarily a tool for testing React apps - you can use Jest with all sorts of projects and we want to make that obvious. You can read more about the ideas behind the re-design in [this issue](https://github.com/facebook/jest/issues/7265).
```

Comment:
is it "redesign" or "re-design"?

Reply:
Just redesign, there's no confusion with another word